## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 422931 `

Logs: https://github.com/mdaniels5757/nixpkgs-review-gha/actions/runs/16099328410


---
### `x86_64-linux` (sandbox = true)
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>pluto</li>
  </ul>
</details>


---
### `aarch64-linux` (sandbox = true)
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>pluto</li>
  </ul>
</details>


---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>pluto</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>pluto</summary>
<pre>Building subPackage ./pkg/discovery-api
Building subPackage ./pkg/finder
Building subPackage ./pkg/helm
Building subPackage ./pkg/kube
buildPhase completed in 1 minutes 46 seconds
Running phase: checkPhase
@nix { "action": "setPhase", "phase": "checkPhase" }
ok  	github.com/fairwindsops/pluto/v5/pkg/api	0.017s
ok  	github.com/fairwindsops/pluto/v5/pkg/discovery-api	0.026s
ok  	github.com/fairwindsops/pluto/v5/pkg/finder	0.016s
W0706 13:12:23.383447   15144 helm.go:133] found duplicate release default/helmtest in a deployed state - this may produce inconsistent results
--- FAIL: TestHelm_getManifest_badClient (0.00s)
    --- FAIL: TestHelm_getManifest_badClient/three_-_bad_client (0.00s)
        helm_test.go:363: 
            	Error Trace:	/private/tmp/nix-build-pluto-5.21.8.drv-0/source/pkg/helm/helm_test.go:363
            	Error:      	"Get \"http://127.0.0.1:9999/api/v1/namespaces\": dial tcp 127.0.0.1:9999: connect: operation not permitted" does not contain "connect: connection refused"
            	Test:       	TestHelm_getManifest_badClient/three_-_bad_client
FAIL
FAIL	github.com/fairwindsops/pluto/v5/pkg/helm	0.044s
FAIL</pre>
</details>
</details>


---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>pluto</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>pluto</summary>
<pre>Building subPackage ./pkg/discovery-api
Building subPackage ./pkg/finder
Building subPackage ./pkg/helm
Building subPackage ./pkg/kube
buildPhase completed in 48 seconds
Running phase: checkPhase
@nix { "action": "setPhase", "phase": "checkPhase" }
ok  	github.com/fairwindsops/pluto/v5/pkg/api	0.012s
ok  	github.com/fairwindsops/pluto/v5/pkg/discovery-api	0.020s
ok  	github.com/fairwindsops/pluto/v5/pkg/finder	0.008s
W0706 13:08:46.455840   11056 helm.go:133] found duplicate release default/helmtest in a deployed state - this may produce inconsistent results
--- FAIL: TestHelm_getManifest_badClient (0.00s)
    --- FAIL: TestHelm_getManifest_badClient/three_-_bad_client (0.00s)
        helm_test.go:363: 
            	Error Trace:	/private/tmp/nix-build-pluto-5.21.8.drv-0/source/pkg/helm/helm_test.go:363
            	Error:      	"Get \"http://127.0.0.1:9999/api/v1/namespaces\": dial tcp 127.0.0.1:9999: connect: operation not permitted" does not contain "connect: connection refused"
            	Test:       	TestHelm_getManifest_badClient/three_-_bad_client
FAIL
FAIL	github.com/fairwindsops/pluto/v5/pkg/helm	0.030s
FAIL</pre>
</details>
</details>

--- a/pkgs/by-name/pl/pluto/package.nix
+++ b/pkgs/by-name/pl/pluto/package.nix
@@ -23,6 +23,8 @@ buildGoModule rec {
     "-X main.version=v${version}"
   ];
 
+  __darwinAllowLocalNetworking = true; # for tests
+
   meta = with lib; {
     homepage = "https://github.com/FairwindsOps/pluto";
     description = "Find deprecated Kubernetes apiVersions";

